.travis.yml

```diff
@@ -1,22 +0,0 @@
-language: node_js
```

Comment:
We don't want to remove this file. Were the tests failing on Travis?

Reply:
No problem, returned it